server/plugin.go

```diff
@@ -64,3 +69,13 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 func (p *Plugin) MessageWillBeUpdated(c *plugin.Context, newPost *model.Post, _ *model.Post) (*model.Post, string) {
 	return p.FilterPost(newPost)
 }
+
+func removeAccents(s string) string {
+	t := transform.Chain(norm.NFD, runes.Remove(runes.In(unicode.Mn)), norm.NFC)
+	output, _, e := transform.String(t, s)
+	if e != nil {
+		panic(e)
```

Comment:
This worries be a bit. Can a user crash the plugin by providing a certain input?

Can the error just be returned and handled properly?

Reply:
As far as I read, the only errors than can be returned are `ErrShortDst` and `ErrShortSrc`, and they should not occur because the memory is allocated in `transform.String()`.

Can we just return the original chain in case of `e != nil` ?

Reply:
I'm also fine with just returning `s` in this case.